map_gui/src/render/road.rs

```diff
@@ -50,14 +52,31 @@ impl DrawRoad {
                 && pair[1].lane_type.is_for_moving_vehicles()
             {
                 let pl = r.shift_from_left_side(width).unwrap();
+                let first_segment_distance = ( pl.length().inner_meters() - (Text::from(&name).rendered_width(prerender)*0.1) ) / 2.0;
```

Comment:
Should this be refactored, i.e, compute the width of the text once?

Reply:
Yes please - see other comment